bnelson777/id.ly
# IL-33: Get bluetooth react native package working

- Added the package 'react-native-ble-plx'

- Allows to scan for bluetooth devices. Can only scan for bluetooth low energy devices, so devices that do not support BLE will not be picked up.

To test:
Find a BLE compatible device and turn its bluetooth on. Open the app and click on "[dev] Bluetooth". The app will start scanning for bluetooth devices and a popup will show the found device(s). If you do not see a popup, then no devices were found.
